administration/src/bp-modules/applications/ApplicationCard.tsx

```diff
@@ -191,29 +168,29 @@ const ButtonsApplicationPending = (props: {
 
 const ButtonsApplicationResolved = (props: {
   applicationStatus: ApplicationStatus | null | undefined // TODO Remove null|undefined once this type is narrowed
-  primaryButtonHref: string | undefined
+  primaryButtonHref: string
   onSecondaryButtonClick: () => void
 }) => {
   const { t } = useTranslation('applicationsOverview')
-
   return (
     <>
       {/* Make the outer Tooltip independent of the button's disabled state */}
       {(props.applicationStatus === ApplicationStatus.Approved ||
         props.applicationStatus === ApplicationStatus.ApprovedCardCreated) && (
         <Tooltip title={props.primaryButtonHref ? undefined : t('incompleteApplicationDataTooltip')}>
           <div>
-            <Button
-              color='primary'
-              variant='contained'
-              disabled={props.primaryButtonHref === undefined}
-              href={props.primaryButtonHref}
-              startIcon={<CreditScore />}>
-              {' '}
-              {props.applicationStatus === ApplicationStatus.ApprovedCardCreated
-                ? t('createCardAgain')
-                : t('createCard')}
-            </Button>
+            <Link to={props.primaryButtonHref}>
```

Comment:
Instead of Wrapping the button with Link we can use component:

```js
  component={Link}
  to={props.primaryButtonHref}
```

Reply:
good find but it will require some custom hover styling if i apply the provided changes.
I think we have lots of such occurrences.
I added it to #2535